Gemfile

```diff
@@ -24,6 +24,8 @@ gem 'jbuilder', '~> 2.0'
 # bundle exec rake doc:rails generates the API under doc/api.
 gem 'sdoc', '~> 0.4.0', group: :doc
 
+gem 'smarter_csv', require: false
```

Comment:
because it infers attribute keys from the first row of the csv file. so then we can do things like attributes[:room_and_capacity], instead of attributes[4], which reads a lot better